client/bomber_sprites.c

```diff
@@ -0,0 +1,180 @@
+#include "enum.h"
+/** SDL.h includes in bomber_sprites.h **/
+#include "bomber_sprites.h"
+#include "map.h"
+#include "base_map_manager.h"
+
+/**
+ * Contain all sprites for bombers and bomb
+**/
+static  SDL_Rect blue_sprites[5][4];
+static  SDL_Rect black_sprites[5][4];
+static  SDL_Rect red_sprites[5][4];
+static  SDL_Rect white_sprites[5][4];
+static  SDL_Rect bomb_sprites[3][3];
+
+/**
+ * Init all statics by launching all init functions
+**/
+void initSprites()
+{
+    initBlueBomberSprites();
+    initBlackBomberSprites();
+    initRedBomberSprites();
+    initWhiteBomberSprites();
```

Comment:
All colors have for the only difference the second argument of the init_rect, couldn't we factorize by defining it on a variable and reuse the code?

Reply:
c'est faux, à moins que le découpage ne soit pas fiable. ^^